docs/source/quickstart.mdx

```diff
@@ -74,6 +78,47 @@ The example files located in `graphql/TheSpaceDevs/` include:
 
     See the [command reference](/apollo-mcp-server/command-reference#config-options) for a list of available configuration options. 
 
+### Docker deployment
+
+1. Modify the configuration YAML for the example code to the following:
+    ```yaml graphql/TheSpaceDevs/config.yaml {7, 10}
+    endpoint: https://thespacedevs-production.up.railway.app/
+    transport:
+      type: streamable_http
+    operations:
+      source: local
+      paths:
+        - /data/operations
+    schema:
+      source: local
+      path: /data/api.graphql
+    overrides:
+      mutation_mode: all
+    introspection:
+      execute:
+        enabled: true
+      introspect:
+        enabled: true
+      search:
+        enabled: true
+    ```
+
+Then, from the root directory of your local repo, run this Docker command to start a local graph with an MCP Server:
+    ```sh showLineNumbers=false
+    docker run \
+      -it --rm \
+      --name apollo-mcp-server \
+      -p 5000:5000 \
+      -v $PWD/graphql/TheSpaceDevs/config.yaml:/config.yaml \
+      -v $PWD/graphql/TheSpaceDevs:/data \
+      ghcr.io/apollographql/apollo-mcp-server:latest /config.yaml
+    ```
+
+    This command:
+    - Starts an MCP Server in a Docker container
+    - Maps configuration files into the proper place for the Apollo MCP Server container
+    - Forwards port 5000 for accessing the MCP Server
```

Comment:
Does the graph also run with this command?

Reply:
Then we should update the previous step since it currently says "run this Docker command to start a local graph with an MCP Server:"

Reply:
Great catch. I completely missed it since that was what I was trying to do when I first wrote the step for the command, but then realized the command I was trying wasn't as demonstrative of what the Docker container for the MCP server could do and then didn't reread the English after my docker command worked. 😖

Anyway, I think it's all good now.